tests/functional/v2/task/testIngestShards.js

```diff
@@ -123,5 +126,23 @@ describe('Test IngestShards', function () {
         );
         assertResults(events, series);
     });
+
+    it('should strip the event uuid during ingestion', async () => {
+        const start = shardFromTimestamp(getTs(-120));
+        const stop = start + 9000000;
+        const events = generateFakeEvents(start, stop, 100);
+
+        await Promise.all(events.map(e => cacheClient.pushMetric(e)));
```

Comment:
might also be able to remove the function wrap here too
```suggestion
        await Promise.all(events.map(cacheClient.pushMetric));
```

Reply:
See above